site/docs/v1/tech/apis/_system-logs-request-body.adoc

```diff
@@ -6,6 +6,11 @@ The format string used to format the date and time columns in the export result.
 +
 When this parameter is omitted a default format of `M/d/yyyy hh:mm:ss a z` will be used. See the https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html[DateTimeFormatter patterns] for additional examples.
 
+[field]#includeArchived# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`#::
+Whether to include rotated, compressed logs in addition to the set of active logs in the export result. The default behavior is to only export the set of logs actively being written.
++
+When this parameter is included with a value of `true`, the `lastNBytes` parameter is ignored and all logs are exported with all log data.
```

Comment:
You might add a note to `lastNBytes` stating that it is ignored when `includeArchived` is `true`.